dsse/sign.go

```diff
@@ -31,6 +31,16 @@ type Envelope struct {
 	Signatures  []Signature `json:"signatures"`
 }
 
+/*
+DecodePayload returns the serialized body, decoded
+from the envelope's payload field. A flexible
+decoder is used, first trying standard base64, then
+URL-encoded base64.
+*/
+func (e *Envelope) DecodePayload() ([]byte, error) {
```

Comment:
should the `DecodePayload` API name indicate it's for base64?

Reply:
perhaps something like `DecodeB64Payload`

Reply:
Done in https://github.com/secure-systems-lab/go-securesystemslib/pull/17/commits/f258f0cdb1d12c40ed0b825fae9ad1308a25d8d2